packages/grpc-native-core/index.js

```diff
@@ -146,6 +146,29 @@ exports.load = function load(filename, format, options) {
   return loadObject(builder.ns, options);
 };
 
+/**
+ * Load a gRPC package definition as a gRPC object hierarchy
+ * @param packageDef grpc~PackageDefinition The package definition object
+ * @return {Object<string, *>} The resulting gRPC object
+ */
+exports.loadPackageDefinition = function loadPackageDefintion(packageDef) {
+  const result = {};
+  for (const serviceFqn in packageDef) {
+    const service = packageDef[serviceFqn];
+    const nameComponents = serviceFqn.split('.');
+    const serviceName = nameComponents[-1];
```

Comment:
Did you mean `nameComponents.slice(-1)[0]`?

Reply:
Yes, something like that. I forgot that wasn't a thing in JavaScript.